miso-web/src/main/webapp/scripts/hot_index.js

```diff
@@ -0,0 +1,71 @@
+HotTarget.index = {
+  getCreateUrl: function() {
+    return Urls.rest.indices.create;
+  },
+  getUpdateUrl: function(id) {
+    return Urls.rest.indices.update(id);
+  },
+  requestConfiguration: function(config, callback) {
+    callback(config)
+  },
+  fixUp: function(index, errorHandler) {
+  },
+  createColumns: function(config, create, data) {
+    return [
+        {
+          header: 'Family',
+          data: 'familyName',
+          readOnly: true,
+          include: true,
+          unpack: function(index, flat, setCellMeta) {
+            flat.familyName = index.family ? index.family.name : config.indexFamily.name;
+          },
+          pack: function(index, flat, errorHandler) {
+            if (!index.family) {
+              index.family = config.indexFamily;
+            }
+          }
+        },
+        HotUtils.makeColumnForInt("Position", true, 'position', HotUtils.validator.integer(true, 1, 2)),
+        HotUtils.makeColumnForText('Name', true, 'name', {
+          description: 'For unique dual index families, the name of index 1 and index 2 must match for automatic selection to work',
+          validator: HotUtils.validator.requiredTextNoSpecialChars
+        }),
+        HotUtils.makeColumnForText(config.indexFamily.fakeSequence ? 'Fake Sequence' : 'Sequence', true, 'sequence', {
```

Comment:
What do you think about changing `Fake Sequence` to `Demultiplexing Sequence`, or something similar/descriptive?

Reply:
Changed to "Demultiplexing Name" since it is a name rather than a sequence